Makefile

```diff
@@ -106,15 +103,9 @@ generate-page: repo/assets
 # Repos
 #
 
-# Clone OCR-D/assets to ./repo/assets
-repo/assets:
-	mkdir -p $(dir $@)
-	git clone https://github.com/OCR-D/assets "$@"
-
-# Clone OCR-D/spec to ./repo/spec
-repo/spec:
-	mkdir -p $(dir $@)
-	git clone https://github.com/OCR-D/spec "$@"
+# Update OCR-D/assets and OCR-D/spec resp.
+repo/assets repo/spec:
+	git submodule update --init
 
 #
 # Spec
```

Comment:
Below here under phony `spec` (required by `install`), still the .yml files are copied into ocrd_validators. This used to leave my working tree with uncommitted changes whenever upstream forgot to upate them. Wouldn't it be better to have symlinks relative to `repo/spec` in ocrd_validators now?

Reply:
Files still need to be copied (symlink would break when installing). However, this copying should not need to happen automatically anymore, but should be called explicitly when there was an actual change to spec.